COLLADAMaya/include/COLLADAMayaPhysXXML.h

```diff
@@ -0,0 +1,1010 @@
+/*
+Copyright (c) 2015 Starbreeze
+
+This file is part of COLLADAMaya.
+
+Portions of the code are:
+Copyright (c) 2005-2007 Feeling Software Inc.
+Copyright (c) 2005-2007 Sony Computer Entertainment America
+Copyright (c) 2004-2005 Alias Systems Corp.
+
+Licensed under the MIT Open Source License,
+for details please see LICENSE file or the website
+http://www.opensource.org/licenses/mit-license.php
+*/
+
+#pragma once
+
+#include "COLLADAMayaPrerequisites.h"
+#include "COLLADASWStreamWriter.h"
+
+#include <libxml/parser.h>
+
+namespace COLLADAMaya
+{
+    namespace PhysXXML
```

Comment:
did you type all this ? or did you take this from another file?
if so, please indicate where this comes from (copyright, license?)